server/chainquery/models/InputModel.js

```diff
@@ -0,0 +1,77 @@
+const getterMethods = {
+  // Add as needed, prefix all methods with `generated`
+}
+
+export default (sequelize, {
+  STRING, BOOLEAN, INTEGER, TEXT, DECIMAL
+}) => sequelize.define(
+  'input',
+  {
+    id: {
+      primaryKey: true,
+      type: INTEGER,
+      set() { },
+    },
+    transaction_id: {
+      type: INTEGER,
+      set() { },
+    },
+    transaction_hash: {
+      type: STRING,
+      set() { },
+    },
+    input_address_id: {
+      type: INTEGER,
+      set() { },
+    },
+    is_coinbase: {
+      type: INTEGER,
```

Comment:
boolean

Reply:
Ha, alright!  I didn't realize `TINYINT(1)` is exactly the same as `BOOL`
https://dev.mysql.com/doc/refman/5.7/en/numeric-type-overview.html
> **BOOL, BOOLEAN**
> These types are synonyms for TINYINT(1). A value of zero is considered false. Nonzero values are considered true